src/oscar/apps/basket/forms.py

```diff
@@ -299,17 +299,22 @@ def cleaned_options(self):
         return options
 
 
-class SimpleAddToBasketForm(AddToBasketForm):
-    """
-    Simplified version of the add to basket form where the quantity is
-    defaulted to 1 and rendered in a hidden widget
-
-    Most of the time, you won't need to override this class. Just change
-    AddToBasketForm to change behaviour in both forms at once.
-    """
-
+class SimpleAddToBasketMixin:
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         if 'quantity' in self.fields:
             self.fields['quantity'].initial = 1
             self.fields['quantity'].widget = forms.HiddenInput()
+
+
+class SimpleAddToBasketForm(SimpleAddToBasketMixin, AddToBasketForm):
+    """
+    Simplified version of the add to basket form where the quantity is
+    defaulted to 1 and rendered in a hidden widget
+
+    If you changed `AddToBasketForm`, you'll need to override this class
+    as well by doing:
+
+    class SimpleAddToBasketForm(SimpleAddToBasketMixin, AddToBasketForm):
+        pass
+    """
```

Comment:
Do we need a `pass` statement here? I'm surprised it doesn't complain without one since there is no other code in the class...

Reply:
Yes, I believe we need it - https://docs.python.org/3/tutorial/classes.html#odds-and-ends.  

I'm surprised too - PyCharm didn't show any syntax violations.

Reply:
OK - please add the `pass` statement.